src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,98 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.regex.Pattern;
+
+/**
+ * A Command stores the command, together with the command string. The appropriate action can be called subsequently
+ * that correspond towards the actions specified.
+ */
+public class Command {
+    /**
+     * The command stored.
+     */
+    private final AvailableCommands CURRENT_COMMAND;
+    /**
+     * The string representation of the command stored.
+     */
+    private final String COMMAND_STRING;
+
+    /**
+     * The list of commands that are available for use.
+     */
+
+    public enum AvailableCommands {
+        EXIT,
+        LIST,
+        MARK,
+        UNMARK,
+        DELETE,
+        ADD_TODO,
+        ADD_DEADLINE,
+        ADD_EVENT,
+        FIND
+    }
```

Comment:
I like the way you use enum for different types of commands! 👍

Reply:
Ok.